.github/workflows/jdbc-tests-pgaudit-enable.yml

```diff
@@ -0,0 +1,191 @@
+name: JDBC Tests With PG AUDIT Enable
+on: [push, pull_request]
+
+jobs:
+  run-babelfish-jdbc-tests-with-pg-audit:
+    env:
+      INSTALL_DIR: psql
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        id: checkout
+
+      - name: Install Dependencies
+        id: install-dependencies
+        if: always()
+        uses: ./.github/composite-actions/install-dependencies
+
+      - name: Build Modified Postgres
+        id: build-modified-postgres
+        if: always() && steps.install-dependencies.outcome == 'success'
+        uses: ./.github/composite-actions/build-modified-postgres
+        with:
+          install_dir: 'psql'
+          code_coverage: 'yes'
+      
+      - name: Compile ANTLR
+        id: compile-antlr
+        if: always() && steps.build-modified-postgres.outcome == 'success'
+        uses: ./.github/composite-actions/compile-antlr
+      
+      - name: Build Extensions
+        id: build-extensions
+        if: always() && steps.compile-antlr.outcome == 'success'
+        uses: ./.github/composite-actions/build-extensions
+
+      - name: Build tds_fdw Extension
+        id: build-tds_fdw-extension
+        if: always() && steps.build-extensions.outcome == 'success'
+        uses: ./.github/composite-actions/build-tds_fdw-extension
+
+      - name: Build vector Extension
+        id: build-vector-extension
+        if: always() &&  steps.build-tds_fdw-extension.outcome == 'success'
+        uses: ./.github/composite-actions/build-vector-extension
+
+      - name: Build PostGIS Extension
+        id: build-postgis-extension
+        if: always() &&  steps.build-vector-extension.outcome == 'success'
+        uses: ./.github/composite-actions/build-postgis-extension
+        
+      - name: Install Extensions
+        id: install-extensions
+        if: always() && steps.build-postgis-extension.outcome == 'success'
+        uses: ./.github/composite-actions/install-extensions
+        with:
+          wal_level: logical
+
+      - name: Build pgaudit Extension
+        uses: ./.github/composite-actions/build-pgaudit-extension
+        with:
+          install_dir: 'psql'
+
+      - name: Ignore some JDBC files 
+        run: |
+          export PATH=~/${{env.INSTALL_DIR}}/bin:$PATH
+          export PG_SRC=~/work/babelfish_extensions/postgresql_modified_for_babelfish
+          export inputFilesPath=${{inputs.input_dir}}
+          cd test
+          sudo sed -i '$a\ignore#!#babel_datatype_sqlvariant' JDBC/jdbc_schedule
+          sudo sed -i '$a\ignore#!#babel_datatype_sqlvariant-vu-prepare' JDBC/jdbc_schedule
+          sudo sed -i '$a\ignore#!#BABEL-2843' JDBC/jdbc_schedule
+          sudo sed -i '$a\ignore#!#babel_datatype_sqlvariant-vu-verify' JDBC/jdbc_schedule
+          sudo sed -i '$a\ignore#!#babel_datatype_sqlvariant-vu-cleanup' JDBC/jdbc_schedule
```

Comment:
Lets order these according to test name and add the JIRA which are tracking these issues.
For `BABEL-2843` we can simply mention why we are ignoring it.